xcode/Subconscious/Shared/Services/DatabaseService.swift

```diff
@@ -1371,9 +1372,27 @@ final class DatabaseService {
         })
         .first
     }
+    
+    func writeActivity<Data: Codable>(event: ActivityEvent<Data>) throws {
```

Comment:
Let's add a test for this method using in-memory SQL

Reply:
Yep 👍